mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/StrategyDependencyResolver.java

```diff
@@ -26,26 +28,46 @@
  */
 class StrategyDependencyResolver {
 
-    protected static final String SMART_TESTING_STRATEGY_PREFIX = "smart.testing.strategy.";
+    protected static final String SMART_TESTING_STRATEGY_PREFIX = Configuration.SMART_TESTING_CUSTOM_STRATEGIES + ".";
 
     private final Path propertiesPath; // TODO this could be configurable through system property and with this we need a path
+    private final String[] customStrategies;
```

Comment:
How about initializing it with empty array?
something like `private final String[] customStrategies = new String[0];`

We did the same way for strategies when we were using system property